.gitignore

```diff
@@ -9,3 +9,4 @@
 
 node_modules/**
 build/**
+coverage/**
```

Comment:
This is created while running `bender run` with coverage plugin enabled. I want to avoid a situation where someone accidentally pushes CC results to the repo.